app/_hub/kong-inc/zipkin/1.0.x.md

```diff
@@ -0,0 +1,108 @@
+    <a href="https://github.com/Kong/kong/blob/master/CHANGELOG.md">CHANGELOG</a>
+    for details.
+  </div>
+
+type: plugin
+categories:
+  - analytics-monitoring
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.0.x
+      - 1.5.x
+      - 1.4.x
+      - 1.3.x
+      - 1.2.x
+      - 1.1.x
+      - 1.0.x
+      - 0.14.x
+  enterprise_edition:
+    compatible:
+      - 1.5.x
+      - 1.3-x
+      - 0.36-x
+      - 0.35-x
+      - 0.34-x
+      - 0.33-x
+      - 0.32-x
+
+params:
+  name: zipkin
+  service_id: true
+  route_id: true
+  consumer_id: true
+  protocols: ['http', 'https', 'tcp', 'tls', 'grpc', 'grpcs']
+  dbless_compatible: yes
+  config:
+    - name: http_endpoint
+      required: true
+      default: ''
+      value_in_examples: http://your.zipkin.collector:9411/api/v2/spans
+      description: |
+        The full HTTP(S) endpoint to which Zipkin spans should be sent by Kong.
+    - name: sample_ratio
+      required: false
+      default: '`0.001`'
+      value_in_examples: 0.001
+      description: |
+        How often to sample requests that do not contain trace ids.
+        Set to `0` to turn sampling off, or to `1` to sample **all** requests.
+    - name: default_service_name
+      required: false
+      description: |
+        The default service name to override the unknown-service-name spans.
+    - name: include_credential
+      required: true
+      default: true
+      value_in_examples: true
+      description: |
+        Should the credential of the currently authenticated consumer be included in metadata sent to the Zipkin server?
+
+---
+
+## How it Works
+
+When enabled, [this plugin](https://github.com/Kong/kong-plugin-zipkin) traces requests in a way compatible with [zipkin](https://zipkin.io/).
+
+The code is structured around an [opentracing](http://opentracing.io/) core using the [opentracing-lua library](https://github.com/Kong/opentracing-lua) to collect timing data of a request in each of Kong's phases.
+The plugin uses opentracing-lua compatible extractor, injector, and reporters to implement Zipkin's protocols.
+
+### Extractor and Injector
+
+An opentracing "extractor" collects information from an incoming request.
+If no trace ID is present in the incoming request, then one is probabilistically generated based on the `sample_ratio` configuration value.
+
+An opentracing "injector" adds trace information to an outgoing request. Currently, the injector is only called for the request proxied by kong; it is **not** yet used for requests to the database or by other plugins (such as the [http-log plugin](./http-log/)).
+
+This plugin follows Zipkin's ["B3" specification](https://github.com/openzipkin/b3-propagation/) as to which HTTP headers to use. Additionally, it supports [Jaegar](http://jaegertracing.io/)-style `uberctx-` headers for propagating [baggage](https://github.com/opentracing/specification/blob/master/specification.md#set-a-baggage-item).
```

Comment:
The support for Jaeger plugin is only partial and should remain indocumented for now.